src/ClientData/include/ClientData/CallstackInfo.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_CALLSTACK_INFO_H_
+#define CLIENT_DATA_CALLSTACK_INFO_H_
+
+#include <stdint.h>
+
+#include "ClientData/CallstackType.h"
+namespace orbit_client_data {
+
+// This class is used on the client to represent a unique callstack, containing of the frames (as
+// pcs) as well as a `CallstackType`.
```

Comment:
```suggestion
// PCs) as well as a `CallstackType`.
```
Or "program counters".